Dockerfile

```diff
@@ -1,8 +1,14 @@
-FROM alpine:3.4
+FROM alpine:3.7
 
+ENV HELM_VERSION=v2.8.2
+ENV HELM_LOCATION="https://kubernetes-helm.storage.googleapis.com"
+ENV HELM_FILENAME="helm-${HELM_VERSION}-linux-amd64.tar.gz"
+ENV HELM_SHA256="614b5ac79de4336b37c9b26d528c6f2b94ee6ccacb94b0f4b8d9583a8dd122d3"
 RUN apk add --update curl && \
-	curl -L https://kubernetes-helm.storage.googleapis.com/helm-v2.0.0-linux-amd64.tar.gz | \
-	tar zxf - && mv /linux-amd64/helm /usr/local/bin/ && rm -r /linux-amd64 && apk del curl
+    curl -L ${HELM_LOCATION}/${HELM_FILENAME} -o ${HELM_FILENAME} && \
```

Comment:
Might be outside of the scope of this PR but you could use the busybox `wget` command that is bundled with the Alpine image and then there would be no need to add and remove curl.

Reply:
Done.